xml/deploy_cephadm.xml

```diff
@@ -1074,23 +1086,65 @@ config: OK
 <screen>&prompt.smaster;ceph-salt apply --non-interactive</screen>
    </tip>
   </sect2>
-
-  <sect2 xml:id="day2-deployment">
-   <title>Deploying Services and Gateways</title>
-   <para>
-    After you have deployed the basic &ceph; cluster, you need to deploy core
-    services to more cluster nodes. To make the cluster data accessible to
-    clients, deploy additional services as well.
-   </para>
-   <para>
-    Currently, we support deployment of &ceph; services on the command line by
-    using the &ceph; orchestrator (<command>ceph orch</command> subcommands).
-   </para>
+  <sect2 xml:id="deploy-min-cluster-final-steps">
+    <title>Final Steps</title>
+    <para>
+      Once the <command>ceph-salt apply</command> command has completed,
+      you should have one MON and one MGR running on the
```

Comment:
Is there a particular reason that sometimes we use an entity for MON and sometimes not?

Reply:
Forgetfulness? But for the same reason why we'd use SES vs. &productname; - the shorthand.